src/components/Signup/Signup.js

```diff
@@ -1,12 +1,16 @@
 import React, { useState } from 'react'
```

Comment:
Can you merge in master please? There shouldn't be CSS anymore, it's been converted to styled components

Reply:
I have added some css in Signup.module.css, won't it break the form?

Reply:
You'll have to convert your css to styled components as well